include/gfx.h

```diff
@@ -81,69 +158,69 @@ Gfx* Gfx_SetFog2(Gfx* gfx, s32 r, s32 g, s32 b, s32 a, s32 n, s32 f);
 Gfx* Gfx_CallSetupDLImpl(Gfx* gfx, u32 i);
 Gfx* Gfx_CallSetupDL(Gfx* gfx, u32 i);
 void Gfx_CallSetupDLAtPtr(Gfx** gfxp, u32 i);
-Gfx* func_8012BFC4(Gfx* gfx);
-Gfx* func_8012BFEC(Gfx* gfx);
-Gfx* func_8012C014(Gfx* gfx);
-void func_8012C058(GraphicsContext* gfxCtx);
-void func_8012C080(GraphicsContext* gfxCtx);
-void func_8012C0A8(GraphicsContext* gfxCtx);
-void func_8012C0D0(GraphicsContext* gfxCtx);
-void func_8012C0F8(GraphicsContext* gfxCtx);
-void func_8012C120(GraphicsContext* gfxCtx);
-void func_8012C148(GraphicsContext* gfxCtx);
-void func_8012C170(GraphicsContext* gfxCtx);
-Gfx* func_8012C198(Gfx* gfx);
-void func_8012C1C0(GraphicsContext* gfxCtx);
-Gfx* func_8012C1EC(Gfx* gfx);
-void func_8012C214(GraphicsContext* gfxCtx);
-void func_8012C240(GraphicsContext* gfxCtx);
+Gfx* Gfx_SetupDL57(Gfx* gfx);
+Gfx* Gfx_SetupDL57_b(Gfx* gfx);
+Gfx* Gfx_SetupDL52_NoCD(Gfx* gfx);
+void Gfx_SetupDL58_Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL57_Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL50_Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL51_Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL52_Xlu(GraphicsContext* gfxCtx);
+void Gfx_SetupDL53_Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL54_Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL55_Xlu(GraphicsContext* gfxCtx);
+Gfx* Gfx_SetupDL26(Gfx* gfx);
+void Gfx_SetupDL26_Opa(GraphicsContext* gfxCtx);
+Gfx* Gfx_SetupDL23(Gfx* gfx);
+void Gfx_SetupDL23_Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL25_Xlu2(GraphicsContext* gfxCtx);
 void func_8012C268(struct GameState* gameState);
```

Comment:
`Gfx_SetupDL25_OpaWrapper` or maybe `Gfx_SetupDL25_OpaState/Gfx_SetupDL25_StateOpa` (not sure best order) since it takes `gameState` perhaps?

Reply:
The reason I didn't name this is because it seems to be a carry-over function from OoT which can be seen here:
https://github.com/zeldaret/oot/blob/5086939880806333a88e714b26a7d720a8dfbb6a/src/code/z_rcp.c#L1010

It uses the same setupDL (25), is used in similar places, but the OoT function has more going on with it. It can also take `PlayState` as it does in OoT, but I decided to leave it for now